report/src/system_evaluation/setup.tex

```diff
@@ -1,17 +1,18 @@
 \section{Experiment design}\label{sec:experiment_design}
 %Forsoegsopstilling, der inkludere en tegning af omraadet set oppefra med indtegning af beacons, objectern og hvor rigtige position for test person.
-Experiments for the evaluation of the implemented system are carried out in an $4.75m \times 6.5m$ room.
+The implemented system's evaluation experiments are conducted in a room measuring $4.75m \times 6.5m$.
 In the room, a cabinet is placed between two tables, effectively dividing the room into two smaller areas. 
 In each of the smaller areas a table with six chairs is placed and on the walls are placed whiteboards.
 \begin{figure}[h]
     \centering
-    \includegraphics[scale=0.7]{images/experiment_room.png}
+    \includegraphics[scale=0.5]{images/experiment_room.png}
     \caption{A sketch of the meeting room in which the experiments were carried out.}
     \label{fig:experiment_room}
 \end{figure}
 On one end of each table is a large, rectangular window.
 Furthermore, one table has a large TV screen placed near its window. 
-Near the door of the room, an airconditioning controller and a large glass window is located.
+Near the door of the room, an airconditioning controller and a large glass window is located. 
+Next to the door is placed a glass pane, acting as a window out into the common area. 
```

Comment:
```suggestion

```

Reply:
This is crucial when commenting on the results in the discussion.